crypto/fipsmodule/bcm.c

```diff
@@ -19,11 +19,17 @@
 #include <openssl/crypto.h>
 
 #include <stdlib.h>
-#if defined(BORINGSSL_FIPS)
+#if defined(BORINGSSL_FIPS) && defined(OPENSSL_ANDROID)
 #include <sys/mman.h>
 #include <unistd.h>
 #endif
 
+#if defined(BORINGSSL_FIPS) && defined(OPENSSL_WINDOWS)
```

Comment:
I think a comment here would be nice, just to explain what's happening with the pragmas below